### DISCUSSION TITLE
[FEATURE] 鼠标点击菜单后，下方的一块区域出现阴影，查看录屏

### DISCUSSION THREAD

**Discussion by ghost** [HEIGE-PCloud/DoIt#305] [General]
### 鼠标点击菜单后，下方的一块区域出现阴影，查看录屏：

https://user-images.githubusercontent.com/29486126/139254407-b20c1a06-b82a-4655-ac66-2d844baa03a6.mov

### 有价值的参考
但是，在该站点中却没有复现：https://planet-cloud.fun
什么问题呢？

**Comment by ghost** [comment]
mac OS 上的三个浏览器都这样。
并且在手机端也复现。

**Comment by HEIGE-PCloud** [comment]
在 https://hugodoit.pages.dev/ 上能复现吗？

**Comment by ghost** [comment]
> 在 https://hugodoit.pages.dev/ 上能复现吗？

这个在mac OS 下三个浏览器，以及手机端 都没问题。

不知道我的版本为什么导致那样的效果。一开始我以为就是这个特效，对比你的以后才发现了问题。

**Comment by HEIGE-PCloud** [comment]
是否对主题做过任何自定义修改？或者请提供一个最小可复现 demo 方便排查。

**Reply by HEIGE-PCloud** [reply]
Convert to a discussion since it is not an issue relate to the theme itself.

**Reply by ghost** [reply]
> 是否对主题做过任何自定义修改？或者请提供一个最小可复现 demo 方便排查。

没有对主题做过任何自定义修改。
我发现原因了：由于开启了 Waline、Vline 评论功能，然后只要加载了一次带有该评论的页面后，整个网站都会出现该问题。

但是我验证了 twikoo，不会出现问题。
可以优化一下吗？
谢谢！

**Comment by HEIGE-PCloud** [comment]
这个动画是因为 Waline 和 Valine 都直接向 header 中写入了自己的 CSS。其中的 `@keyframes pulse` 覆盖了 [animate.css](https://animate.style/) 中 pulse 的动画。

https://github.com/walinejs/waline/blob/8e6b4464849d7d5f2e08f2723c1f8e134f6630b2/packages/client/src/styles/animation.scss#L11

解决方法就是别用 Waline 了。

**Reply by lizheming** [reply]
不好意思，`@waline/client@1.3.11` 最新版里已经移除了这些 keyframes 定义了，感谢你帮忙排查~

**Reply by ghost** [reply]
我不懂前端。
但是这几天我通过大家学习一些前端的内容，谢谢大家的解答和帮助！
